GUI/GUIListPanel.h

```diff
@@ -50,7 +50,8 @@ class GUIListPanel :
         MouseEnter, // Mouse left the panel
         MouseLeave, // Mouse left the panel
         DoubleClick,// Double click
-        KeyDown    // Key Down
+		KeyDown,    // Key Down
+		EdgeHit //!< Tried scrolling the selection past the first or last item. data = 0 for top edge, data = 1 for bottom edge.
```

Comment:
It might not be bad to use -1 for top edge and 0 for bottom edge. You could simplify even more the scrolling handling stuff by just sending the scroll direction when you send data.
Maybe this has other ramifications I'm missing though.